reaktive/src/jsMain/kotlin/com/badoo/reaktive/utils/ThrowableExt.kt

```diff
@@ -1,18 +1,5 @@
 package com.badoo.reaktive.utils
 
 actual fun Throwable.printStack() {
-    println(stackTrace)
-    var inner: Throwable? = cause
-    while (inner != null) {
-        println("Caused by: ${inner.stackTrace}")
-        inner = inner.cause
-    }
-}
-
-private val Throwable.stackTrace: String
-    get() =
-        try {
-            asDynamic().stack.toString()
-        } catch (ignored: Throwable) {
-            toString()
-        }
\ No newline at end of file
+    console.log(this)
```

Comment:
maybe let's use console.error(this)?

Reply:
I wanted to make it consistent with other platforms. Regular printStackTrace prints via regular output in regular color. Our uncaught error handlers are printing via error.

Reply:
@lukaville Fixed error printing, via error output